doctrine/annotations/1.0/config/routes/annotations.yaml

```diff
@@ -1,3 +1,3 @@
 controllers:
-    resource: ../src/Controller/
+    resource: ../../src/Controller/
```

Comment:
Couldn't we be using the following instead: 

```
controllers:
    resource: "%kernel.project_dir%/src/Controller"
```
???

Reply:
was my first idea, but parameters are not parsed in the config area, would be a whole new feature

Reply:
Ah, okey. Thanks

Reply:
As the related change is done in 3.4, it means that this config will break on project using 3.3, right?

Reply:
right, so I'll rebase against 3.3 as bug fix!

Reply:
they won't: the fallback mechanism will look on the previous path and work as before

Reply:
ok, great then.

Reply:
IIRC, this was what we had at first and it didn't work.